### ISSUE TITLE
How could I install MaxEntScan?

### ISSUE THREAD

**Issue by goshng** [frarei312/ExceS-A-An-Exon-Centric-Split-Aligner#2]
MaxEntScan is said to be required for ExceS-A. I googled for MaxEntScan, but it was hard to find it. Is there any link to MaxEntScan?

Thank you!

**Comment by goshng** [comment]
I found one like this:

http://hollywood.mit.edu/burgelab/maxent/Xmaxentscan_scoreseq.html

**Comment by frarei312** [comment]
Yes, that is the right one. You can download it from this website.
